backend/models/User.js

```diff
@@ -1,19 +1,39 @@
 const mongoose = require("mongoose");
-const User = new mongoose.Schema({
+const bcrypt = require('bcrypt');
+
+const UserSchema = new mongoose.Schema({
   firstName: {
     type: String,
-    required: true,
+    required: [true, "Please enter your first name"],
+    lowercase: true,
```

Comment:
`lowercase` saves everything as lowercase - makes managing the database easier as well as using text on the front end more consistent.